carlsim/test/compartments.cpp

```diff
@@ -174,6 +174,7 @@ TEST(COMPARTMENTS, spikeTimesCPUvsData) {
 *
 * This test makes sure that CPU mode and GPU mode of the compartment model produce the exact same spike times,
 * even with RK4 at 50 time steps.
+* FIX ME: TEST CURRENTLY FAILS AT numIntSteps = 30 (or 40 depending on machine) with small spike time deviation (2ms)
```

Comment:
This is not acceptable. Please don't modify test case parameter just for passing